devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy

```diff
@@ -42,8 +42,15 @@ metadata {
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FFFF", outClusters: "0019", manufacturer: "Megaman", model: "ZLL-ExtendedColor", deviceJoinName: "INGENIUM Light" //INGENIUM ZB RGBW Light
 
 		// Innr
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "innr", model: "RB 185 C", deviceJoinName: "Innr Light", mnmn: "SmartThings", vid: "generic-rgbw-color-bulb-2000K-6500K" //Innr Smart Bulb Color
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "innr", model: "FL 130 C", deviceJoinName: "Innr Light" //Innr Flex Light Color
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "innr", model: "RB 185 C", deviceJoinName: "Innr Smart Bulb Colour", mnmn: "SmartThings", vid: "generic-rgbw-color-bulb-2000K-6500K" //Innr Smart Bulb Colour
```

Comment:
- Please add Raw Descriptions in a comments above each fingerprint
// Raw Description ....
fingerprint 
- Please correct spacing (use 1 tab instead of 4 spaces), it may be useful to edit it in code editor first and then paste and commit the result.
- According to new fingerprint writing rules -  deviceJoinName for all Innr bulbs should look like: deviceJoinName: "Innr Light"
-  According to new fingerprint writing rules -  profileId, inClusters and outClusters are not needed.